lib/error.js

```diff
@@ -13,10 +13,20 @@ function FormError(key, options, req, res) {
             return options.message || this.getMessage(key, options, req, res);
         }
     });
+    Object.defineProperty(this, 'title', {
+        enumerable: true,
+        get: function () {
+            return options.title || this.getTitle();
```

Comment:
Shall I add those arguments? Tbh I am not terribly keen, but happy to comply if needed.